deploy/jbossas/kit/jboss-wf/org/modeshape/index-provider/lucene/main/module.xml

```diff
@@ -18,10 +18,15 @@
 <module xmlns="urn:jboss:module:1.3" name="org.modeshape.index-provider.lucene">
     <resources>
         <resource-root path="modeshape-lucene-index-provider-${project.version}.jar" />
+        <resource-root path="lucene-analyzers-common-${version.org.apache.lucene}.jar" />
+        <resource-root path="lucene-core-${version.org.apache.lucene}.jar" />
+        <resource-root path="lucene-queries-${version.org.apache.lucene}.jar" />
+        <resource-root path="lucene-queryparser-${version.org.apache.lucene}.jar" />
+        <resource-root path="lucene-sandbox-${version.org.apache.lucene}.jar" />
+        <resource-root path="lucene-backward-codecs-${version.org.apache.lucene}.jar" />
```

Comment:
I understand why we'd want to remove the higher-level dependency on Lucene and instead have that dependency restricted to the Lucene index provider, but why are we _embedding_ the Lucene libraries inside the provider module rather than simply having the provider module depend on a separate Lucene module?